nilearn/plotting/img_plotting.py

```diff
@@ -603,12 +603,53 @@ def plot_epi(epi_img=None, cut_coords=None, output_file=None,
     return display
 
 
+def _plot_roi_contours(display, roi_img, cmap, alpha, linewidths):
+    """Helper function for plotting regions of interest ROIs in contours.
+
+    Parameters
+    ----------
+    display : nilearn.plotting.displays.OrthoSlicer, object
+        An object with background image on which contours are shown.
+
+    roi_img : Niimg-like object
+        See http://nilearn.github.io/manipulating_images/input_output.html
+        The ROI/mask image, it could be binary mask or an atlas or ROIs
+        with integer values.
+
+    cmap : matplotlib colormap
+        The colormap for the atlas maps
+
+    alpha : float between 0 and 1
+        Alpha sets the transparency of the color inside the filled
+        contours.
+
+    linewidths : float
+        This option can be used to set the boundary thickness of the
+        contours.
+    """
+    roi_img = _utils.check_niimg_3d(roi_img)
+    roi_data = roi_img.get_data()
+    labels = np.unique(roi_data)
+    cmap = plt.cm.get_cmap(cmap)
+    color_list = cmap(np.linspace(0, 1, len(labels)))
+    for idx, label in enumerate(labels):
+        if label == 0:
+            continue
+        data = (roi_data == label)
+        data = data.astype(np.int)
+        img = new_img_like(roi_img, data, affine=roi_img.affine)
+        display.add_contours(img, levels=[0.5], colors=[color_list[idx - 1]],
```

Comment:
What would be the good default ```levels``` threshold for labels based ROIs ?

Reply:
I think that the one that you used (.5) is good.